src/main/java/com/google/sps/servlets/SignInServlet.java

```diff
@@ -37,7 +37,7 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
       json = "{ \"status\": true, \"url\": \"" + signOutUrl + "\" }";
     }
     else {
-      String signInUrl = userService.createLoginURL("/user-list-test.html");
+      String signInUrl = userService.createLoginURL("/sign-in-validity-check");
       json = "{ \"status\": false, \"url\": \"" + signInUrl + "\" }";
```

Comment:
I think tommy mentioned something about this not following the styling guide suggestions but correct me if i am wrong :0